htdocs/install/mysql/migration/21.0.0-22.0.0.sql

```diff
@@ -246,3 +246,9 @@ ALTER TABLE llx_contratdet DROP COLUMN remise;
 ALTER TABLE llx_extrafields ADD COLUMN aiprompt text;
 
 ALTER TABLE llx_menu ADD COLUMN showtopmenuinframe integer DEFAULT 0;
+
+-- Add contacts to reccurent / templates invoices / factures
```

Comment:
When you need to change the database structure, you must first push an independent PT with just the database structure addition, with no code change.